From e9131465dd01cff4d46aa5bf6a6ec2066a408bd6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 12 Sep 2019 13:41:17 +0200
Subject: [PATCH 088/152] ci: Fail on any common lint errors

Now that we fixed all pre-existing errors that are common between
regular and legacy configuration, we can stop filtering the result
by the lines modified by the merge request.

This will allow us to catch errors in merge requests that slipped
through until now, for example when leaving a newly-unused import
behind.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/716
---
 .gitlab-ci/run-eslint.sh | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/.gitlab-ci/run-eslint.sh b/.gitlab-ci/run-eslint.sh
index 82642d4f0..aad95f46a 100755
--- a/.gitlab-ci/run-eslint.sh
+++ b/.gitlab-ci/run-eslint.sh
@@ -86,9 +86,13 @@ run_eslint LEGACY
 echo Done.
 create_common
 
+if ! is_empty $OUTPUT_FINAL; then
+  cat $OUTPUT_FINAL
+  exit 1
+fi
+
 # Just show the report and succeed when not testing a MR
 if [ -z "$CI_MERGE_REQUEST_TARGET_BRANCH_NAME" ]; then
-  cat $OUTPUT_FINAL
   exit 0
 fi
 
-- 
2.23.0

